code/model/ForumThread.php

```diff
@@ -302,7 +302,7 @@ static function notify(Post $post) {
 					$email = new Email();
 					$email->setFrom(Email::getAdminEmail());
 					$email->setTo($member->Email);
-					$email->setSubject('New reply for ' . $post->Title);
+					$email->setSubject(_t('Post.NEWREPLY', 'New reply for') . " " . $post->Title);
```

Comment:
This should use the ability to pass parameters to the translation string rather than just pushing it onto the end.

eg: `'New reply for {title}', array('title' => $post->Title)`